tasks/friends-app.md

```diff
@@ -0,0 +1,61 @@
+[![MIT Licensed][icon-mit]][license]
+[![Awesome][icon-awesome]][awesome]
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+[![Telegram][icon-chat]][chat]
+
+# Friends App
+
+When you're done with your challenge push your code to github repo and publish
+it using `gh-pages`. _Add link to it to your main frontend tasks repo._
+
+If you honestly finished all the previous steps than go ahead and share it with
+others - send a message in [course channel](chat) with the link to you repo and the words "Weather APP - finished" and add link to repo with code of the
+website.
+
+## Friends app
+
+Create a tiny social friends' search page with users cards, search, sorting and filtering of them by age, name or whatewer you want with [Random User API](https://randomuser.me) :)
+![image](https://i.imgur.com/5tcpqcY.png).
+
+### As a user, I would like to:
+
+* See list of my mock friends :)
```

Comment:
Maybe you can add what can be in this list? I believe this will make this part more descriptive

Reply:
done